apps/tangle-dapp/src/containers/bridge/BridgeContainer.tsx

```diff
@@ -315,26 +318,29 @@ const BridgeContainer = () => {
   ]);
 
   const routerFeeDetails: FeeDetailProps | null = useMemo(() => {
-    if (!routerQuote) {
+    if (
+      !routerQuote ||
+      !selectedToken ||
+      !amount ||
+      !routerQuote?.bridgeFee.amount
+    ) {
       return null;
     }
 
-    const sendingAmount = parseFloat(
-      formatEther(BigInt(amount?.toString() ?? '0')),
-    );
+    const sendingAmount = parseFloat(formatEther(BigInt(amount.toString())));
 
     const formattedSendingAmount =
       sendingAmount.toString() + ' ' + routerQuote?.bridgeFee.symbol;
```

Comment:
Check for `undefined` values here to prevent displaying an undefined string in the UI.